content/_index.md

```diff
@@ -27,13 +58,11 @@ url= "https://www.linuxfoundation.org/press/linux-foundation-launches-open-sourc
 
 +++
 
+## Documentation
+
 Valkey is an open source (BSD) high-performance key/value datastore that supports a variety of workloads such as **caching**, **message queues**, and can act as a **primary database**.
 Valkey can run as either a **standalone** daemon or in a **cluster**, with options for **replication** and **high availability**.
 
 Valkey natively supports a rich collection of datatypes, including **strings**, **numbers**, **hashes**, **lists**, **sets**, **sorted sets**, **bitmaps**, **hyperloglogs** and more.
 You can operate on data structures in-place with an expressive collection of commands.
 Valkey also supports native extensibility with built-in scripting support for **Lua** and supports **module** plugins to create new commands, data types, and more.
```

Comment:
```suggestion
Valkey can run as either a **standalone** daemon or in a **cluster**, with options for **replication** and **high availability**. Valkey natively supports a rich collection of datatypes, including **strings**, **numbers**, **hashes**, **lists**, **sets**, **sorted sets**, **bitmaps**, **hyperloglogs** and more.
You can operate on data structures in-place with an expressive collection of commands.
Valkey also supports native extensibility with built-in scripting support for **Lua** and supports **module** plugins to create new commands, data types, and more.
```

Because I pulled a sentence earlier, just removing that sentence and compacting these two together.